tabular/tests/unittests/test_hyperband_for_decision_trees.py

```diff
@@ -0,0 +1,101 @@
+from autogluon.tabular import TabularPrediction as task
+
+from autogluon.tabular.learner.default_learner import DefaultLearner as Learner
+from autogluon.tabular.features.generators.auto_ml_pipeline import AutoMLPipelineFeatureGenerator
+
+from autogluon.tabular.models.catboost.catboost_model import CatBoostModel
+from autogluon.tabular.models.lgb.lgb_model import LGBModel
+from autogluon.tabular.models.rf.rf_model import RFModel
+from autogluon.tabular.models.xt.xt_model import XTModel
+
+
+from autogluon.core.space import Int
+from autogluon.core.metrics import accuracy
+
+from autogluon.core.scheduler.hyperband import HyperbandScheduler
+from autogluon.core.task.base.base_task import compile_scheduler_options
+
+from autogluon.tabular.utils import infer_problem_type
+
+def hyperband_hpo(model_name, label_column='class', strategy="bayesopt",
```

Comment:
shouldn't strategy be `"bayesopt_hyperband"`?

Reply:
to turn off hyperband the current method is simply set epoch>1. The searcher uses either bayesopt or random without any additional changes.